### ISSUE TITLE
CSS class today is not added if min option is set to current time

### ISSUE THREAD

**Issue by Djules** [chrisdavies/tiny-date-picker#79]
Using option `min: new Date()` prevents CSS class 'today' to be added to current day cell.

It seems to occur because the render method of day-picker is using current time instead of current day (with hours, minutes, seconds, milliseconds to zero).

So for now, we should use current day without time as `min` value.
```js
var today = new Date();
today.setHours(0, 0, 0, 0);

// then …
{
  min: today
}
```

**Comment by chrisdavies** [comment]
Hm. Good find.

**Comment by chrisdavies** [comment]
Should be fixed in the latest release (3.2.5). Thanks!
